public/js/pieces/algobots.js

```diff
@@ -0,0 +1,2728 @@
+let a = parseInt(tokenData.hash.slice(0, 16));
+var c = parseInt(tokenData.tokenId.slice(4)).toString();
+
+function z() {
+     return a ^= a << 13, a ^= a >> 17, a ^= a << 5, (a < 0 ? 1 + ~a : a) % 1e3
+}
+Number.prototype.map = function(a) {
+     return Math.round(this * a / 999)
+};
+var e = document.createElement("style");
+e.innerText = "@media (orientation: portrait) {#stina {width: 100vw;display: block;margin: 0 auto;}}@media (orientation: landscape) {#stina {height: 100vh;display: block;margin: 0 auto;}}", document.head.appendChild(e);
+var M = ["#A964CF", "#08B5FC", "#00D3C3", "#FFCE63", "#FF931E", "#EF56A3", "#f95555", "#6969ff", "#666666", "#CCCCCC"];
+const l = {
```

Comment:
I'm fairly certain this is where the SVG image data is embedded, although I'm not familiar with the format used here.